src/darwin/CHIPTool/CHIPTool.xcodeproj/project.pbxproj

```diff
@@ -565,7 +565,6 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = CHIPTool/CHIPTool.entitlements;
-				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
Revert file?

Reply:
Yeah this was accidental removal. I'll revert the file.